knative-operator/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY . .
 ENV GOFLAGS="-mod=vendor"
 RUN go build -o /tmp/operator ${BASE}/knative-operator/cmd/manager
 
-FROM registry.ci.openshift.org/ocp/ubi-minimal:8
+FROM registry.access.redhat.com/ubi8-minimal:latest
```

Comment:
We only need to be aware, that we need then to update openshift/release too to update the references: https://github.com/openshift/release/blob/53169552f7d1733e3ce3b553f982e3fca1197c6f/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__415.yaml#L42

Reply:
> We only need to be aware, that we need then to update openshift/release too to update the references: https://github.com/openshift/release/blob/53169552f7d1733e3ce3b553f982e3fca1197c6f/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__415.yaml#L42

I think there's an automation on openshift/release in place that actually optimize `FROM` and replaces public RHT registry to CI registry. 

Moreover as a consequence of pulling images from`registry.redhat.io` (in other Dockerfiles) will require auth token to execute `make images`. That's a concer towards local builds, CI probably can make it work either way.

Reply:
We had that before. But [this PR](https://github.com/openshift-knative/serverless-operator/pull/2473) changed that to using CI registry. Let's see if this works in the long term.

Reply:
You can use registry.access.redhat.com here as long as this is merged: https://github.com/openshift-knative/hack/pull/158
It will re-generate the CI config so that the registry.access.redhat.com ubi image is replaced in OpenShift CI runs with the correct ImageStream. 
OTOH, Konflux will be using directly the image from registry.access.redhat.com, AFAIK.

Reply:
PR for update in main: #2693